app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>RubyTms</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```

Comment:
Nháy đôi và dùng I18n nè
-> chú to anh luân review là  thấy liền nè, đỡ việc đợi tới thứ 2 mới mợt -> rút  kinh nghiệm bé